tests/view_tests/tests/test_debug.py

```diff
@@ -557,6 +559,21 @@ def __str__(self):
         text = reporter.get_traceback_text()
         self.assertIn('USER: [unable to retrieve the current user]', text)
 
+    @override_settings(ROOT_URLCONF='view_tests.urls')
```

Comment:
I guess it's not intentional to decorate bot the class and the method with this? (test seems to be passing without both).